plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -44,6 +46,7 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 	value,
 	onChange,
 	productId,
+	attributeType = 'regular',
```

Comment:
I'm not sure that it is best to default this param vs forcing it to be passed in. I did this for now to avoid having to change tests that weren't passing this prop in.

Reply:
I'm in favor of removing `attributeType` altogether.  It forces this component to contain more knowledge about the consumer than it should.